### PR TITLE
Upgrade sail from 1.5.1 to 3.2.4

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description

I've tried to take over https://github.com/thepracticaldev/dev.to/pull/4136 made by dependabot, but I couldn't push to it, so I created a separate branch.

I've also ran `rails g sail:update` to update the tables schema and enabled `/sail` for tech admins.

What's missing is to follow these instructions to save settings from the production DB if we have any: https://github.com/vinistock/sail#populating-the-database and commit that file.

### CODE REVIEW

**Comment by rhymes** [comment]
@maestromac we should make a decision about this... :D

File: Gemfile [hunk 1]
```diff
@@ -80,7 +80,7 @@ gem "rolify", "~> 5.2" # Very simple Roles library
 gem "rouge", "~> 3.12" # A pure-ruby code highlighter
 gem "rubyzip", "~> 1.2", ">= 1.3.0" # Rubyzip is a ruby library for reading and writing zip files
 gem "s3_direct_upload", "~> 0.1" # Direct Upload to Amazon S3
-gem "sail", "~> 1.5" # Sail is a lightweight Rails engine that brings an admin panel for managing configuration settings on a live Rails app
+gem "sail", "~> 3.2" # Sail is a lightweight Rails engine that brings an admin panel for managing configuration settings on a live Rails app
```
Comment (mstruve) [inline Gemfile]: What config settings do we use this to manage?

Comment (maestromac) [reply Gemfile]: Sail isn't fully in place yet so there's no way to direclty configure it, ie we haven't [mount the route](https://github.com/vinistock/sail#installation).

**Comment by maestromac** [comment]
This would be superseded by https://github.com/thepracticaldev/dev.to/pull/4729

**Comment by rhymes** [comment]
Now that https://github.com/thepracticaldev/dev.to/pull/4729 is in, we don't need to upgrade anything.

**Pull request closed without merging**